lib/oidcstrategy.js

```diff
@@ -159,6 +203,10 @@ function Strategy(options, verify) {
   // validator will throw exception if a required option is missing
   var validator = new Validator(validatorConfiguration);
   validator.validate(itemsToValidate);
+
```

Comment:
validatorConfiguration and the validate are closely related, I think they should stay together without the line break.